npc/quests/quests_moscovia.txt

```diff
@@ -8884,10 +8885,11 @@ mosk_fild02,124,202,3	script	Lusalka#rus23	4_F_RUSGREEN,{
 		end;
 	}
 	mes "-When Lusalka watches you, you are blacked out-";
+	close2;
 	specialeffect2 EF_FLASHER;
 	percentheal -100,0;
 	donpcevent "Lusalka#rus23::OnDisable";
-	close;
+	end;;
```

Comment:
remove extra `;`

and maybe `unitkill getcharid(3);` would be better/suitable for killing a char? 
Perhaps would be helpful in future code standardization?